core/src/main/scala/org/apache/spark/eventhubs/utils/SimulatedCachedReceiver.scala

```diff
@@ -0,0 +1,29 @@
+package org.apache.spark.eventhubs.utils
```

Comment:
Could you add the apache header to all these new files?

Reply:
Good point, I've added the headers.